src/makeCalculator.js

```diff
@@ -36,8 +35,41 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculatorObject = {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
```

Comment:
all methods should have only one number as a parameter , 
`add(number)`
and in this case `add` to whatever we have as a `result `is 
somthing like this 
`calculator.result += number;`

Reply:
This has been updated. Let me know if anything else is needed. Thanks